CHANGELOG.md

```diff
@@ -16,6 +16,20 @@ should change the heading of the (upcoming) version to include a major version b
 
 -->
 
+# 5.6.1
```

Comment:
You are adding a new feature so let's bump the version
```suggestion
# 5.7.0
```

Reply:
Done